lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -119,19 +119,14 @@ public static function get_fonts_dir() {
 	 *
 	 *     @type string $path    Path to the directory.
 	 *     @type string $url     URL for the directory.
-	 *     @type string $subdir  Sub-directory of the directory.
-	 *     @type string $basedir Base directory.
-	 *     @type string $baseurl Base URL.
 	 * }
 	 * @return array Modified upload directory.
 	 */
 	public static function set_upload_dir( $defaults ) {
-		$defaults['basedir'] = WP_CONTENT_DIR;
-		$defaults['baseurl'] = content_url();
-		$defaults['subdir']  = '/fonts';
```

Comment:
Is anything lost by not setting basedir, baseurl, and subdir here? It seems like unexpected things could happen if some of the properties for `uploads_dir` are changed and some are not, but I haven't used this filter before.

Reply:
Based on the definition of [wp_upload_dir](https://developer.wordpress.org/reference/functions/wp_upload_dir/#source), when `path` is not set, basedir + subdir are used, and when `url` is not set, baseurl + subdir are used. I think we are not loosing anything here.

Reply:
Alternatively, we could switch to `basedir` + `subdir` where basedir becomes the path set by the variable `WP_FONT_DIR` and subdir will be the site_id (in case of multi site) and empty for simple sites.
[Please refer this](https://github.com/WordPress/gutenberg/pull/57044/files#diff-c8ed3eebde903fce6544c7e49065ed2f0b5653be1cf936c1b32ea5bab7bff67eR117-R119)